api/src/v1/university/models/faculty_model.py

```diff
@@ -1,24 +1,11 @@
 from typing import List
-
 from pydantic import BaseModel, RootModel
 
-from shared.src.enums import FacultyEnum
-from shared.src.tables import FacultyTable, FacultyTranslationTable
-
 
 class Faculty(BaseModel):
-    id: FacultyEnum
+    id: int
     title: str
```

Comment:
I would prefer this naming for the DTO.
```suggestion
    name: str
```

Reply:
Okay changed the field from title to name and therefore also the response object.